.yarnrc.yml

```diff
@@ -12,3 +12,7 @@ plugins:
     spec: "https://go.mskelton.dev/yarn-outdated/v4"
 
 yarnPath: .yarn/releases/yarn-4.1.1.cjs
+
+supportedArchitectures:
+  os: [current, darwin, linux]
```

Comment:
Do we also need to support windows for non-devs?

Reply:
May be worth adding as it has come up before?